internal/website/docs/next/reference/hooks/usePost.mdx

```diff
@@ -50,7 +50,7 @@ export default function Page() {
   const { usePost } = client;
   const post = usePost({
     id: 'hello-world',
-    idType: PostIdType.SLUG,
+    idType: PostIdType.URI,
```

Comment:
Does this part need to change? `PostIdType.SLUG` is a valid `idType` and `hello-world` _seems_ like a SLUG in the context of the other examples on the page. Neither way is technically wrong, but keeping SLUG might help avoid some confusion around what is a URI vs. what is a SLUG.